db/sqitch.plan

```diff
@@ -78,3 +78,6 @@ tables/application_form_data_001_add_rls_for_analyst 2022-10-24T19:51:34Z Anthon
 tables/form_data_002_add_permisions_for_analyst 2022-10-24T20:30:23Z Anthony Bushara <anthony@button.is> # Add permissions for analyst
 computed_columns/application_organization_name 2022-10-25T15:05:18Z Anthony Bushara <anthony@button.is> # computed column to display the organization name for analysts
 computed_columns/application_form_data [computed_columns/application_form_data@1.9.0] 2022-10-25T18:34:11Z Anthony Bushara <anthony@button.is> # Add ccbc_analyst permission
+@1.12.0 2022-10-28T18:19:06Z ,,, Vladimir <vladimir@button.is> # rollback summary_view changes
```

Comment:
why is there a release in here?

For this PR you can update `computed_columns/application_status.sql` directly since it's still unreleased, and you can do a rework of `views/summary_view`. `views/summary_view.sql` is where you'd drop the view. You should not edit the reworked deploy files (i.e. any file ending with `@1.*.*.sql`)

Reply:
Ok, I will try to update computed_columns/application_status.sql directly. I believe I already tried and got errors from sqitch .. but maybe those were caused by some other my mistake. :)